docs/diagrams/ListEvContactSequenceDiagram.puml

```diff
@@ -0,0 +1,77 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":ListEvContactCommandParser" as ListEvContactCommandParser LOGIC_COLOR
+participant "l:ListEvContactCommand" as ListEvContactCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "pr:EventSetContainsEventPredicate" as EventSetContainsEventPredicate MODEL_COLOR
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("listevcontact 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("listevcontact 1")
+activate AddressBookParser
+
+create ListEvContactCommandParser
+AddressBookParser -> ListEvContactCommandParser
+activate ListEvContactCommandParser
+
+ListEvContactCommandParser --> AddressBookParser
+deactivate ListEvContactCommandParser
+
+AddressBookParser -> ListEvContactCommandParser : parse("1")
+activate ListEvContactCommandParser
+
+create ListEvContactCommand
+ListEvContactCommandParser -> ListEvContactCommand
+activate ListEvContactCommand
+
+ListEvContactCommand --> ListEvContactCommandParser : l
+deactivate ListEvContactCommand
+
+ListEvContactCommandParser --> AddressBookParser : l
+deactivate ListEvContactCommandParser
```

Comment:
Actually is it possible to terminate the dotted lines until the object deletion symbol?

Reply:
Also, if object deletion is included for `ListEvCommandParaser`, do you think it would be better to include for `AddressBookParser` too?